doc/uftrace.html

```diff
@@ -247,6 +247,7 @@
 ...     perf_event: [ .green[on]  ] - perf (PMU) event support
 ...       schedule: [ .green[on]  ] - scheduler event support
 ...       capstone: [ .red[OFF] ] - full dynamic tracing support
+...      libunwind: [ .red[OFF] ] - stacktrace support support (optional for debugging)
```

Comment:
It's not required here. Please read the previous discussion at  https://github.com/namhyung/uftrace/pull/1744#discussion_r1252985345.